charts/backstage/README.md

```diff
@@ -236,6 +237,19 @@ Now that the ConfigMap has been created on your Kubernetes cluster, you can refe
 
 The chart will mount the content of the ConfigMap as a new `app-config.extra.yaml` file and automatically pass the extra configuration to your instance.
 
+### Pass configuration to be stored in a ConfigMap
+
+Instead of following the previus step (`Pass extra configuration files`), you can get the Config Map automatically deployed with this Helm Chart by defining the key `appConfig`:
```

Comment:
`s/Instead of/In addition to/` since it's not exclusive, both options are possible.

Reply:
Fixed! And added warning about priority